src/transformers/tokenization_utils.py

```diff
@@ -24,11 +24,12 @@
 import re
 from collections import UserDict, defaultdict
 from contextlib import contextmanager
-from typing import List, Optional, Sequence, Tuple, Union
+from typing import Any, Dict, List, NamedTuple, Optional, Sequence, Tuple, Union
 
-from tokenizers import AddedToken, Encoding
-from tokenizers.decoders import Decoder
-from tokenizers.implementations import BaseTokenizer
+from tokenizers import AddedToken as AddedTokenFast
+from tokenizers import Encoding as EncodingFast
+from tokenizers.decoders import Decoder as DecoderFast
+from tokenizers.implementations import BaseTokenizer as BaseTokenizerFast
```

Comment:
Post-fixing all the classes imported from `tokenizers` with `Fast` for clear separation

Reply:
If Python tokenizers are on their way to deprecation maybe we should name things the other way around? (Slow or Python for the non-Fast ones)

Reply:
We don't have external classes for the python tokenizers (and our user-facing API is currently `XXXTokenizerFase` also)

Overall, I don't really think when we'll be able to fully deprecate the python tokenizers before the summer (sentencepice support is still quite ahead)